datacats/environment.py

```diff
@@ -301,6 +310,9 @@ def require_data(self):
             raise DatacatsError('Environment datadir damaged. '
                                 'Try "datacats purge" followed by'
                                 ' "datacats init".')
+        if not self.volumes_exist():
+            raise DatacatsError('Volumes damaged. '
+                                'Try "datacats purge" followed by "datacats init"')
```

Comment:
A little friendlier and more cautious, like:
`Volume containers could not be found. To reset and discard all data use "datacats purge" followed by "datacats init"`